rtmbot.py

```diff
@@ -153,26 +161,71 @@ def parse_args():
     )
     return parser.parse_args()
 
+def flag_is_true(flag_str):
+    """ Checks if a string flag should evaluate to True. """
+    flag_str = flag_str.strip().lower()
+    return flag_str == 'true'
```

Comment:
I don't know of any common usage here. I generally use "0" and "1" but this looks good to me.

I suppose you could do something like this, but maybe it's overkill:

```python
def flag_is_true(flag_str, truths=None):
     """ Checks if a string flag should evaluate to True. """
     truths = ['true', '1']
     flag_str = flag_str.strip().lower()
     return flag_str in truths
```